doc/whats_new.rst

```diff
@@ -32,6 +32,9 @@ Changelog
 - Add ability to record voltages and synaptic currents from all sections in :class:`~hnn_core.CellResponse`,
   by `Nick Tolley`_ in :gh:`502`.
 
+- Add ability to check instance type of Dipole in :func:`~hnn_core.viz.plot_dipole`,
```

Comment:
this is more of a bugfix I would say ... but this will not get rendered properly. Did you check how to build the documentation in the contributing guide:

https://jonescompneurolab.github.io/hnn-core/dev/contributing.html

Reply:
Sure I will put it in bugfix and check the rendering properly

Reply:
Hey @jasmainak do I need to add my name or profile somewhere to get correct linking from whats_new.html.
For others it shows the correct hyperlink. For me it shows Rajat Partani_

Reply:
Yes, look at the bottom of the page. See also: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages

Reply:
Hey @jasmainak I made all changes. Please check them once